contracts/interfaces/draft-IERC6909.sol

```diff
@@ -49,7 +49,8 @@ interface IERC6909 is IERC165 {
     function isOperator(address owner, address spender) external view returns (bool);
 
     /**
-     * @dev Sets an approval to `spender` for `amount` tokens of type `id` from the caller's tokens.
+     * @dev Sets an approval to `spender` for `amount` of tokens of type `id` from the caller's tokens. An `amount` of
```

Comment:
`amount` is a number, like 1, 17 or 42. 

I would not say Sets an approval to spender for 17 of tokens.